library/src/main/java/com/chuckerteam/chucker/internal/ui/transaction/TransactionPayloadFragment.kt

```diff
@@ -113,7 +142,13 @@ internal class TransactionPayloadFragment :
             }
         }
 
-        menu.findItem(R.id.encode_url).isVisible = false
+        menu.findItem(R.id.encode_url).apply {
+            isVisible = true
+            setOnMenuItemClickListener {
```

Comment:
This options item already had click listener in `TransactionActivity` and called exactly the same fun from ViewModel.